src/libraries/System.Private.CoreLib/src/System/Environment.UnixOrBrowser.cs

```diff
@@ -69,5 +70,36 @@ private static int CheckedSysConf(Interop.Sys.SysConfName name)
             }
             return (int)result;
         }
+
+        /// <summary>
+        /// Get the CPU usage, including the process time spent running the application code, the process time spent running the operating system code,
+        /// and the total time spent running both the application and operating system code.
+        /// </summary>
+        [SupportedOSPlatform("maccatalyst")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        [UnsupportedOSPlatform("browser")]
+        public static ProcessCpuUsage CpuUsage
+        {
+            get
+            {
+                Interop.Sys.ProcessCpuInformation cpuInfo = default;
+                Interop.Sys.GetCpuUtilization(ref cpuInfo);
+
+                ulong userTime100Nanoseconds = cpuInfo.lastRecordedUserTime / 100; // nanoseconds to 100-nanoseconds
+                if (userTime100Nanoseconds > long.MaxValue)
+                {
+                    userTime100Nanoseconds = long.MaxValue;
+                }
```

Comment:
Nit: you could simplify these with ulong.Min

Reply:
Did you mean `Math.Min`?

Reply:
I meant ulong.Min, but Math.Min would work, too.

Reply:
I was not aware of `ulong.Min` exist. Looking at it, I saw it is using `Math.Min` so I used `Math.Min`.